### ISSUE TITLE
How to train reversed model for RL model

### ISSUE THREAD

**Issue by bobbercheng** [brianhuang1019/RL-Chatbot#10]
You mentioned "
When training with policy gradient (pg)

you may need a reversed model

the reversed model is also trained by cornell movie-dialogs dataset, but with source and target reversed.
"
Except downloading pre-trained reversed model, could you please tell how to rain it?

Thank you a lot.

**Comment by brianhuang1019** [comment]
For instance, if a sample in training set is ('How is the weather?', 'It's sunny.').
To train a reversed model, you need to reverse the data into ('It's sunny.', 'How is the weather?').
That means you need to predict the former sentence using the later sentence.
